c/src/nnstreamer-capi-util.c

```diff
@@ -1323,8 +1323,8 @@ ml_get_nnfw_type_by_subplugin_name (const char *name)
  * @brief Checks the availability of the given execution environments.
```

Comment:
minor: `... with custom option.` is omitted

Reply:
updated